Clearpath moose

fabienrohrer opened the pull request:
- [x] Moose PROTO
- [x] Move uneven_terrain.wbt and adapt it for the moose.
    - [x] Improve the rendering of this world.
    - [x] Adapt the controller
- [x] Doc
    - [x] Moose page: ~https://cyberbotics.com/doc/guide/moose?version=feature-moose~ https://cyberbotics.com/doc/guide/moose?version=develop
    - [x] Movie
        - [ ] Publish the movie
    - [x] Robot component
    - [x] Icon
- [x] Log

fabienrohrer commented:
Movie candidate:

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/kFioks3wy7g/0.jpg)](https://www.youtube.com/watch?v=kFioks3wy7g)

fabienrohrer commented:
Are you ok with this movie? It's shows the current state of this simulation.

omichel commented:
Yes, it looks very nice to me (even though I would have loved to see the box slip a little bit on the top of the robot 😄)

fabienrohrer commented:
The robot is also slipping at the beginning of the movie, and I think adding a start/end title would be better.

DavidMansolino commented:
> The robot is also slipping at the beginning of the movie, and I think adding a start/end title would be better.

Also the last 'in-place' rotation seems a bit 'kinematic' (whereas all the other motions look very realistic!).

fabienrohrer commented:
New movie candidate:

- I smoothed the path
- I fixed the in-place rotation
- I added robot pauses
- I added titles
- I added an awesome music
- I give a try about to decrease the cardboard frictions, but didn't get good results, It was giving the feeling that the simulation was not robust.

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/joPAnZcOouc/0.jpg)](https://www.youtube.com/watch?v=joPAnZcOouc)

DavidMansolino commented:
> New movie candidate

Very nice !

fabienrohrer commented:
test me

cyberbotics-jenkins commented:
Build Passed :ok_hand:
You can safely merge 'feature-moose' into 'develop'.

fabienrohrer commented:
👍

omichel commented:
:eyes:

omichel commented on docs/guide/moose.md:
```diff
@@ -0,0 +1,42 @@
+
+%robot moose images/robots/moose/model.png
+
+The [Clearpath's Moose robot](https://www.clearpathrobotics.com/moose-ugv/) is a large all-terrain unmanned ground vehicle yet. It can handle tough environments with its rugged build, low ground pressure, and 8x8 traction tires, which allow effortless mobility through soft soils, vegetation, thick muds, and steep grades.
```
```suggestion
The [Moose robot](https://www.clearpathrobotics.com/moose-ugv/) from [Clearpath Robotics](https://www.clearpathrobotics.com) is a large all-terrain unmanned ground vehicle yet.
It can handle tough environments with its rugged build, low ground pressure, and 8x8 traction tires, which allow effortless mobility through soft soils, vegetation, thick muds, and steep grades.
```

omichel commented on docs/guide/moose.md:
```diff
@@ -0,0 +1,42 @@
+#### Moose Field Summary
+
+- `color`: Defines the bumper color.
+- `bodySlot`: Extends the robot with new nodes in the base slot.
```
```suggestion
- `bodySlot`: Extends the robot with new nodes.
```

omichel commented on projects/robots/clearpath/moose/protos/Moose.proto:
```diff
@@ -0,0 +1,427 @@
+# license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+# license url: https://cyberbotics.com/webots_assets_license
+# documentation url: https://www.cyberbotics.com/doc/guide/moose
+# The Clearpath Moose robot  is a large all-terrain unmanned ground vehicle yet.
```
```suggestion
# The Clearpath Moose robot is a large all-terrain unmanned ground vehicle.
```

fabienrohrer commented:
Thank you, I applied your suggestions as-is.

fabienrohrer commented:
test me

cyberbotics-jenkins commented:
Build Passed :ok_hand:
You can safely merge 'feature-moose' into 'develop'.
